components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint/src/main/webapp/consent.jsp

```diff
@@ -20,6 +20,7 @@
 <%@ page import="org.wso2.carbon.identity.application.authentication.endpoint.util.Constants" %>
 <%@include file="localize.jsp" %>
 <%@include file="init-url.jsp" %>
+<%@ page import="java.io.File"%>
```

Comment:
Move page import before includes on all the pages

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/2426/commits/37e4813fd5f8e9078546c990764edd8d9132b13e